gradle.properties

```diff
@@ -30,12 +30,12 @@ POM_DEVELOPER_NAME=facebook
 
 GRADLE_VERSIONS_PLUGIN_VERSION=0.15.0
 
-KOTLIN_VERSION=1.9.10
-DOKKA_VERSION=1.9.10
+KOTLIN_VERSION=1.9.20
+DOKKA_VERSION=1.9.20
 DATA_CLASS_GENERATE_VERSION=1.0.1
 
 # Deps for publishing
-MAVEN_PUBLISH_PLUGIN=0.15.1
```

Comment:
I think there was an issue with some newer version of this plugin so we reverted back to an old one: https://github.com/facebook/litho/commit/ff08c11f6c8869f6a5f8e36d00e7f0d0713aeabe
Mode details about the issue: https://github.com/facebook/litho/issues/978

Reply:
Hmm, that's interesting. It seems though as if the old version is no longer able to publish anything now. It times out which looks like there are some problems with the TLS config. https://github.com/facebook/litho/actions/runs/8635428192/job/23756165085

Reply:
We were trying 0.22 and you're updating it to 0.28 so I think it may be worth checking if publishing with 0.28 isn't losing some of the classes. WDYT?

Reply:
Sure! I can push a snapshot locally and if you or someone else could help validate if it's complete, that would be fab.

Reply:
I guess we could try to use something like this maybe: https://github.com/JakeWharton/diffuse - build a local snapshot using old and new version of the maven publish plugin and compare the apks.